src/mixins/helpers.js

```diff
@@ -110,7 +111,7 @@ var helpers = {
           animating: false
         });
         if (this.props.afterChange) {
-          this.props.afterChange(currentSlide);
+          this.props.afterChange(targetSlide);
```

Comment:
@akiran I had only this line in originalPR, then i kept working on same branch and forgot that i created PR. sorry